server/tests/fetchRetry.test.ts

```diff
@@ -49,7 +49,11 @@ describe("VespaClient", () => {
   }))
 
   beforeAll(() => {
-    vespaClient = new VespaClient(endpoint)
+    vespaClient = new VespaClient(undefined, {
+      vespaMaxRetryAttempts: 3,
+      vespaRetryDelay: 100,
+      queryEndpoint: endpoint,
```

Comment:
these fields are flat @Himanshvarma , is typescript not catching this?

<img width="525" height="478" alt="Screenshot 2025-11-15 at 02 28 02" src="https://github.com/user-attachments/assets/db06a902-9506-439d-932f-7222c8d0ac06" />

Reply:
@Himanshvarma what about feed endpoint? That needs to be sent too